src/combinatorics/configuration.lean

```diff
@@ -435,11 +435,12 @@ begin
   classical,
   have h1 : fintype.card {q // q ≠ p} + 1 = fintype.card P,
   { apply (eq_tsub_iff_add_eq_of_le (nat.succ_le_of_lt (fintype.card_pos_iff.mpr ⟨p⟩))).mp,
-    convert (fintype.card_subtype_compl).trans (congr_arg _ (fintype.card_subtype_eq p)) },
+    convert (fintype.card_subtype_compl _).trans (congr_arg _ (fintype.card_subtype_eq p)) },
   have h2 : ∀ l : {l : L // p ∈ l}, fintype.card {q // q ∈ l.1 ∧ q ≠ p} = order P L,
   { intro l,
     rw [←fintype.card_congr (equiv.subtype_subtype_equiv_subtype_inter _ _),
         fintype.card_subtype_compl, ←nat.card_eq_fintype_card],
+    any_goals { apply_instance },
```

Comment:
Similarly what happened here?

Reply:
This one is because I changed `card_subtype_compl` from
```lean
lemma fintype.card_subtype_compl [fintype α] {p : α → Prop} [decidable_pred p] :
  fintype.card {x // ¬ p x} = fintype.card α - fintype.card {x // p x} :=
```
to
```lean
lemma fintype.card_subtype_compl [fintype α]
  (p : α → Prop) [fintype {x // p x}] [fintype {x // ¬ p x}] :
  fintype.card {x // ¬ p x} = fintype.card α - fintype.card {x // p x} 
```
and the rewrite is somehow not able to find the third fintype instance. If you give it the predicate directly, then it works fine. In the commit I pushed, I modified it to give the predicate directly.

I'm not sure what the best thing to do here is, and I don't really understand what's causing the instance lookup failure... (Maybe it's `ne` vs not-equals?)